user_guide/data_management_grids.rst

```diff
@@ -0,0 +1,306 @@
+.. _user-guide-ui-components-grids:
+    
+Grids
+=====
+
+Initially, a grid is an aggregated view of all the records of the same entity. Each row of a grid is one records, each 
+column is one of the grid properties, and thus each cell displays a value of a property for a record. .
+
+For example, you can open a grid of all the Contacts. 
+
+|
+
+.. image:: ./img/data_management/grid/grid_01.png
+
+|
+
+In the example, you can immediately see some major details of our contacts, such as their first and last names, email, 
+phone number, etc.
```

Comment:
Edit: In this example, you can immediately see some major details of our contacts, such as their first and last names, email, phone number, etc.